src/templates/ja/2020/contributors.html

```diff
@@ -2,7 +2,7 @@
 
 {% block title %}{{ year }} 貢献者 | HTTP ArchiveによるWeb Almanac{% endblock %}
 
-{% block description %}The {{ config.contributors.items() | length }} アナリスト、著者、デザイナー、開発者、編集者、レビュアー、翻訳者として{ year }}年のWeb Almanacに貢献した人々。{% endblock %}
+{% block description %} {{ config.contributors.items() | length }} アナリスト、著者、デザイナー、開発者、編集者、レビュアー、翻訳者として{ year }}年のWeb Almanacに貢献した人々。{% endblock %}
```

Comment:
Nit, but ca we remove the unnecessary space?
```suggestion
{% block description %}{{ config.contributors.items() | length }} アナリスト、著者、デザイナー、開発者、編集者、レビュアー、翻訳者として{ year }}年のWeb Almanacに貢献した人々。{% endblock %}
```